lib/init/grass.py

```diff
@@ -172,13 +173,6 @@ def try_remove(path):
         pass
 
 
-def try_rmdir(path):
```

Comment:
I guess the idea was to have something like in `grass.script.utils` and write `try_rmdir(tmpdir)` instead of `lambda: shutil.rmtree(tmpdir, ignore_errors=True)`, but I don't see clear rule here (brevity vs std library use). As for removing the Cleaner class, I agree, code was much simplified, no need for it anymore (see my fixes for the multiple/incorrect tmp dirs).